### PR TITLE
Jumpstart ag - deployment readme with option for azd

### PR DESCRIPTION
**Pull request by dkirby-ms**
- Adds instructions for azd in deployment readme
- Adds cleanup section and placeholder screenshots

### CODE REVIEW

File: docs/azure_jumpstart_ag/contoso_supermarket/deployment/_index.md [hunk 1]
```diff
@@ -214,6 +214,20 @@ Once automation is complete, users can immediately start enjoying the Contoso Su
 
     > **NOTE: If you see any failure in the deployment, please check the [troubleshooting guide](https://placeholder).**
 
+## Deployment via Azure Developer CLI (experimental)
+
+Jumpstart Agora provides an experimental feature that allows users to deploy with the [Azure Developer CLI](https://learn.microsoft.com/en-us/azure/developer/azure-developer-cli/overview). Follow these steps to try this experimental feature in your subscription.
+
+- Follow to install guide for the [Azure Developer CLI](https://learn.microsoft.com/en-us/azure/developer/azure-developer-cli/install-azd?tabs=winget-windows%2Cbrew-mac%2Cscript-linux&pivots=os-linux) for your environment.
```
Comment (charris-msft) [inline docs/azure_jumpstart_ag/contoso_supermarket/deployment/_index.md:221]: " - Follow **_the_** install guide..."

**Review by charris-msft — APPROVED** [review]
LGTM!

File: docs/azure_jumpstart_ag/contoso_supermarket/deployment/_index.md (hunk 1 above)
Comment (likamrat) [inline docs/azure_jumpstart_ag/contoso_supermarket/deployment/_index.md:221]:
```suggestion
- Follow the install guide for the [Azure Developer CLI](https://learn.microsoft.com/en-us/azure/developer/azure-developer-cli/install-azd?tabs=winget-windows%2Cbrew-mac%2Cscript-linux&pivots=os-linux) for your environment.
```

File: docs/azure_jumpstart_ag/contoso_supermarket/deployment/_index.md [hunk 2]
```diff
@@ -214,6 +214,20 @@ Once automation is complete, users can immediately start enjoying the Contoso Su
 
     > **NOTE: If you see any failure in the deployment, please check the [troubleshooting guide](https://placeholder).**
 
+## Deployment via Azure Developer CLI (experimental)
+
+Jumpstart Agora provides an experimental feature that allows users to deploy with the [Azure Developer CLI](https://learn.microsoft.com/en-us/azure/developer/azure-developer-cli/overview). Follow these steps to try this experimental feature in your subscription.
```
Comment (likamrat) [inline docs/azure_jumpstart_ag/contoso_supermarket/deployment/_index.md:219]:
```suggestion
Jumpstart Agora provides an experimental feature that allows users to deploy with the [Azure Developer CLI](https://learn.microsoft.com/azure/developer/azure-developer-cli/overview). Follow these steps to try this experimental feature in your subscription.
```

**Pull request merged**